packages/kbn-shared-ux-components/src/page_template/solution_nav/solution_nav.tsx

```diff
@@ -181,7 +181,7 @@ export const KibanaPageTemplateSolutionNav: FunctionComponent<
           isCollapsible={canBeCollapsed}
           initialIsOpen={false}
         >
-          <EuiPanel color="transparent" paddingSize="m">
+          <EuiPanel color="transparent" paddingSize="s">
```

Comment:
@cchaos just wondering if we want to change the padding to from `m` to `s`? I think originally the padding was `m` on the collapsbile nav, so it would be smaller with this change?

Reply:
Yes, this is on purpose because of the visual alignment: 
![image](https://user-images.githubusercontent.com/549577/172876237-8961e876-6420-47f2-bb71-2c12c4eb3e16.png)